src/OpenTelemetry/Metrics/MetricPoint.cs

```diff
@@ -1313,15 +1005,18 @@ private void UpdateBase2ExponentialHistogram(double number, ReadOnlySpan<KeyValu
             histogram.Record(number);
         }
 
-        this.OfferExemplarIfSampled(number, tags, isSampled);
-
         this.mpComponents.ReleaseLock();
+
+        this.CompleteUpdate();
+
+        this.UpdateExemplar(number, tags, offerExemplar);
     }
 
-    private void UpdateBase2ExponentialHistogramWithMinMax(double number, ReadOnlySpan<KeyValuePair<string, object?>> tags = default, bool isSampled = false)
+    private void UpdateBase2ExponentialHistogramWithMinMax(double number, ReadOnlySpan<KeyValuePair<string, object?>> tags = default, bool offerExemplar = false)
     {
         if (number < 0)
         {
+            this.CompleteUpdateWithoutMeasurement();
```

Comment:
bug fix? do we need changelog for this?

Reply:
You think it is worth mentioning? Before for measurements of `< 0` against an exponential histogram we no-op the update but we still set `MetricPointStatus = MetricPointStatus.CollectPending` which would cause the `MetricPoint` to be collected during the next export. Not sure if that is actually a bug or just odd 🤣 For cumulative I don't think it would mean anything. For delta, maybe?

/cc @alanwest

Reply:
Pretty subtle and not very noteworthy. If it were just me, I'd skip the changelog, but if you feel differently, I'll volunteer some verbiage:

> Fixed an issue where a zero-valued histogram metric point may be exported after recording a negative measurement. This could only occur when using the `Base2ExponentialHistogram` aggregation. Negative measurements against an exponential histogram are not supported and should be ignored.

Reply:
Since, it does not have any substantial end user impact. I guess ok to skip it.

I pointed this out as I noticed the change looked unrelated to the PR.